src/Univapay/Enums/OnlineBrand.php

```diff
@@ -13,12 +13,19 @@ public static function WE_CHAT_ONLINE() { return self::create(); }
     public static function ALIPAY_CHINA() { return self::create(); }
     public static function ALIPAY_HK() { return self::create(); }
     public static function B_KASH() { return self::create(); }
+    public static function BOOST() { return self::create(); }
```

Comment:
Is `BOOST` new one as I don't see this wallet in the `NewWallets20231115` xlsx?

Reply:
It seems we already have `BOOST` in GPP API codebase. I think you just added here.

Reply:
Its something that was missing previously. See the OnlineBrand class on the API?